src/node/bash_install_node.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
```

Comment:
Can you rename the file to `src/node/install_nvm_with_retries.sh`?

Reply:
sure